### PR TITLE
exclude hotspot for classTestHelper compilation

### PR DESCRIPTION
**Pull request by OscarQQ**
related: #13001
[ci skip]

Signed-off-by: Yixin Qian <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
Grinder/16417  hotspot jdk8 utils and utils2 don't get compiled

classTestHelper is also used at https://github.com/eclipse-openj9/openj9/blob/d0a0f901987eb2ab2c007f3196fd6e6a66604315/test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml#L47 
but since [playlist.xml](https://github.com/eclipse-openj9/openj9/blob/master/test/functional/cmdLineTests/shareClassTests/SCCMLTests/playlist.xml) will only run ibm/openj9 version. So no further action is needed for shareClassTests/SCCMLTests

**Comment by llxia** [comment]
Could you also test JDK17 hotspot? Thanks

File: test/functional/cmdLineTests/utils/build.xml [hunk 1]
```diff
@@ -81,6 +81,13 @@
 	</target>
 
 	<target name="build" >
-		<antcall target="clean" inheritall="true" />
+		<if>
+			<not>
+				<equals arg1="${JDK_IMPL}" arg2="hotspot"  />
```
Comment (llxia) [inline test/functional/cmdLineTests/utils/build.xml:86]: I would prefer to explicitly list ibm/openj9. The same as line 89.

**Comment by llxia** [comment]
run aqa --sdk_resource nightly --build_list functional --target testSCCMLTests2 --jdk_version 16 --jdk_impl hotspot --platform x86-64_linux

**Comment by llxia** [comment]
run aqa -h

**Comment by llxia** [comment]
run aqa --sdk_resource nightly --build_list functional --target testSCCMLTests2 --jdk_version 11 --jdk_impl hotspot --platform x86-64_mac

**Comment by llxia** [comment]
run aqa --sdk_resource nightly --build_list functional --target testSCCMLTests2 --jdk_version 11 --jdk_impl openj9 --platform x86-64_windows

**Comment by llxia** [comment]
run aqa --sdk_resource customized --build_list functional --target testSCCMLTests2 --jdk_version 18 --jdk_impl hotspot --platform x86-64_linux --customized_sdk_url https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk/job/jdk-linux-x64-hotspot/lastSuccessfulBuild/artifact/workspace/target/OpenJDK-jdk_x64_linux_hotspot_2021-06-19-23-30.tar.gz

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**